### PR TITLE
Coordination minutes 2023-01-19

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2023/2023-01-19-coordination.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+### Pre-CHEP Workshop
+
+We have confirmed with WLCG colleagues and LHC computing coordinators the two topics for a [pre-CHEP workshop](https://indico.cern.ch/e/wlcg-hsf23) on 6-7 May 2023 in Norfolk, VA.
+
+- Analysis Facilities
+  - Based on the findings of the AFF group (which we asked to draw conclusions after 12-18 months of work).
+  - How do workloads evolve? How do sites adapt their offerings? What are the benchmarks of success?
+- Non-x86 Computing
+  - Increasingly diverse computing architectures at sites, both in CPU flavours (ARM, PowerPC) and accelartors (GPUs, mainly, from different stables).
```
Comment (amete) [inline organization/_posts/2023/2023-01-19-coordination.md:30]:
```suggestion
  - Increasingly diverse computing architectures at sites, both in CPU flavours (ARM, PowerPC) and accelerators (GPUs, mainly, from different stables).
```

File: organization/_posts/2023/2023-01-19-coordination.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+With that in place we signed the [letter of cooperation](https://docs.google.com/document/d/1wnrbIw4Dv_VgWOleBRkbRddaFAtQ2wRnmqUMrftfnBg/edit?usp=share_link) (in the very standardised NSF format!).
+
+### Google Summer of Code, 2023
+
+Google Summer of Code for 2023 is starting!
+
+This year Benedikt will head up the organising team. After many years Andrei and Antoine step down. *Huge thanks to them for their work!*
+
+**We therefore urgently need two people to join Benedikt at the helm of GSoC for the HSF.** This is a very valuable service for the community and it much appreciated by all of us, so please consider volunteering.
```
Comment (amete) [inline organization/_posts/2023/2023-01-19-coordination.md:49]:
```suggestion
**We therefore urgently need two people to join Benedikt at the helm of GSoC for the HSF.** This is a very valuable service for the community and it is much appreciated by all of us, so please consider volunteering.
```

File: organization/_posts/2023/2023-01-19-coordination.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+## Working Group Updates
+
+### General
+
+We are proceeding well with WG convenerships for 2023. A few WGs are not quite concluded yet (Data Analysis and Software Trigger and Reco), but should converge soon.
+
+We are very grateful to all the conveners, continuing, joining or stepping down, for their contribution to the community through the HSF. **Thank you!** We hope that outgoing conveners remain active in the groups, even if relieved of specific responsibilities.
+
+Today we shall review 2022 and discuss plans for 2023 with 4 of the groups. The other WGs we shall cover next meeting.
```
Comment (amete) [inline organization/_posts/2023/2023-01-19-coordination.md:63]:
```suggestion
Today we shall review 2022 and discuss plans for 2023 with 4 of the groups. The other WGs we shall cover at the next meeting.
```

File: organization/_posts/2023/2023-01-19-coordination.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+#### General HSF Activities
+
+See slides attached to the [agenda](https://indico.cern.ch/event/1225007/).
+
+#### Meetings
+
+*Reminder:* Please try and book meetings in Indico at least 2 weeks in advance!
+
+That way they go into the calendar early and they will be included in the weekly email announcement that goes to HSF Forum.
```
Comment (amete) [inline organization/_posts/2023/2023-01-19-coordination.md:73]:
```suggestion
That way they go into the calendar early and they will be included in the weekly email announcement that goes to the HSF Forum.
```

File: organization/_posts/2023/2023-01-19-coordination.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+- Markus Diefenthaler, EIC (Continues)
+- Efe Yazgan, CMS (Continues)
+- Phil Ilten, LHCb (Joins)
+- Josh McFayden, ATLAS (Steps down)
+
+See slides attached to the [agenda](https://indico.cern.ch/event/1225007/).
+
+Standards would be very interesting. No detailed discussions yet on APIs or specific needs. This came out of an MC4EIC discussion, where specific nuclear generators are used, but they lack many important features found in generators like Sherpa and MG5 (matching, merging, ...).
+
+Workshop on computational efficiency in generators would be welcome again. CMS meeting that discussed this, so there is interest. But how can we engage them better.
```
Comment (amete) [inline organization/_posts/2023/2023-01-19-coordination.md:137]:
```suggestion
Workshop on computational efficiency in generators would be welcome again. CMS meeting that discussed this, so there is interest. But how can we engage them better?
```

**Review by amete — APPROVED** [review]
Apologies for not being able to attend the first coordination meeting of the year @graeme-a-stewart. The minutes look good to me. I only have a few minor editorial comments. Many thanks.

**Comment by graeme-a-stewart** [comment]
Thanks @amete! I could swear I thought that you were there - I don't usually ask people to review if the were not present. All the improvements much appreciated!

**Pull request merged**